src/app/organizations/collection/collection.component.html

```diff
@@ -113,19 +114,32 @@ <h5>
                       </span>
                     </div>
                   </div>
-                </a>
-                <hr />
-                <div fxLayout="row" fxLayoutAlign="space-between">
-                  <span class="update-time"> Last updated {{ entry.dbUpdateDate | date }} </span>
-                  <a
-                    [routerLink]="[
-                      '/' + (entry.entryType === 'tool' ? 'container' : 'workflow') + 's/',
-                      entry.entryPath + (entry.versionName ? ':' + entry.versionName : '')
-                    ]"
-                  >
+                  <hr />
+                  <div fxLayout="row" fxLayoutAlign="space-between center">
+                    <div fxLayout="row" fxLayoutAlign="space-between center" fxLayoutGap="1rem">
+                      <span class="update-time"> Last updated {{ entry.dbUpdateDate | date }} </span>
+                      <div fxLayout="row wrap" fxLayoutGap="1rem" class="labels">
+                        <span
+                          [ngClass]="entry.entryType == 'workflow' ? 'workflow-label-chip' : 'container-label-chip'"
+                          *ngFor="let label of entry.labels"
+                        >
+                          {{ label }}
+                        </span>
+                        <mat-chip
+                          [ngClass]="entry.entryType == 'workflow' ? 'workflow-label-chip' : 'container-label-chip'"
+                          *ngIf="entry.verified == true"
```

Comment:
Make triple equals or get rid of the equals and right-side of the equation entirely.

Reply:
Was working with Java and C in the past few days so I completely forgot triple equals exist haha